manifests/devicepluginC-ds.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: device-plugin-c-ds
+spec:
+  selector:
+      matchLabels:
+        name: device-plugin-c
+  template:
+    metadata:
+      labels:
+        name: device-plugin-c
+    spec:
+      hostNetwork: true
+      containers:
+      - name: device-plugin-a-container
+        image: quay.io/aperevalov/device-plugin:latest
```

Comment:
Maybe we should think about setting up a `k8stopologyawareschedwg` quay repo that all the members of the working group can push too? That would provide us the ability to store images pertaining to all the relevant components, sample-device-plugin, scheduler plugin, NFD, RTE etc. WDYT?

Reply:
great idea, I've try, but only quay.io/my_account/ is available for me.

Reply:
good idea indeed. Will work on it